conans/test/generators/cmake_test.py

```diff
@@ -262,3 +265,24 @@ def settings_are_generated_tests(self):
         self.assertIn('set(CONAN_SETTINGS_COMPILER_VERSION "12")', cmake_lines)
         self.assertIn('set(CONAN_SETTINGS_COMPILER_RUNTIME "MD")', cmake_lines)
         self.assertIn('set(CONAN_SETTINGS_OS "Windows")', cmake_lines)
+
+    def no_output_test(self):
+        client = TestClient()
+        client.run("new Test/1.0 --sources")
+        cmakelists_path = os.path.join(client.current_folder, "src", "CMakeLists.txt")
+        replace_in_file(cmakelists_path,
+                        "conan_basic_setup()",
+                        "set(CONAN_CMAKE_SILENT_OUTPUT True)\nconan_basic_setup()",
+                        output=client.user_io.out)
+        client.run("create . danimtb/testing")
+        self.assertNotIn("Conan: Using cmake global configuration", client.out)
```

Comment:
Please test also the positive case, where the messages are being printed. It might happen that they are not tested elsewhere, and for some bug, they would always be completely silenced.